src/postcss/list.css

```diff
@@ -1,5 +1,5 @@
 .intellyo-list {
-  & .is-dragging {
+  &--is-dragging {
```

Comment:
is it the `.intellyo-list` which has an additinal class `intellyo-drag-sort--is-dragging`? Isn't it the `.intellyo-drag-sort`?

Reply:
in deed, thanks, I fixed.